docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -314,7 +314,7 @@ For securing the state file further, you can also enable access logging on the S
 
 ## Part 6: Terraform Configuration Structure
 
-There are many different approaches to setup your Terraform code structure. Although there is no one prescribed approach, whatever approach you take needs to be catered for the needs of your particular project. Keeping that in mind; the file structure for [Terraform beyond Layer0 example](https://github.com/quintilesims/layer0-examples/blob/master/terraform-beyond-layer0/example-1) is as below:
+There are many different approaches to setup your Terraform code structure. Although there is no one prescribed approach, whatever approach you take needs to be catered for the needs of your particular project. Keeping that in mind; the file structure for [Terraform beyond Layer0 example](https://github.com/quintilesims/guides/blob/master/terraform-beyond-layer0/example-1) is as below:
```

Comment:
We should be giving the users a preferred approach here and explaining why we did what we did rather than hinting at all the different approaches available. A common critique of our guides has been that we don't offer enough trusted opinion and/or best practices in places like this.